packages/neon/neon/lib/src/widgets/cached_image.dart

```diff
@@ -61,8 +66,20 @@ class NeonCachedImage extends StatefulWidget {
   final Color? svgColor;
   final Color? iconColor;
 
-  static Future<Uint8List> _getImageFromUrl(final String url) async {
-    final file = await _cacheManager.getSingleFile(url);
+  final Widget? Function(BuildContext, dynamic)? errorBuilder;
```

Comment:
Separate typedef; perhaps `ErrorWidgetBuilder`.

This could also be in the NeonException as other widgets might also need to use it.